src/systems/tracked-controls.js

```diff
@@ -13,12 +13,13 @@ module.exports.System = registerSystem('tracked-controls', {
     this.lastControllersUpdate = 0;
     // Throttle the (renamed) tick handler to minimum 10ms interval.
     this.tick = utils.throttle(this.throttledTick, 10, this);
-    if (!navigator.getVRDisplays) { return; }
-    navigator.getVRDisplays().then(function (displays) {
-      if (displays.length > 0) {
-        self.vrDisplay = displays[0];
-      }
-    });
+    if (navigator.getVRDisplays) {
+      this.sceneEl.addEventListener('enter-vr', function () {
```

Comment:
I don't know.  for rift/vive, the tracked controllers currently don't make sense outside vr.  but more generally, no reason not to have tracked controllers unassociated with VR HMD, Should those get a user height offset just because the Rift and Vive controllers do while in VR?